Documentation/ceph-csi-drivers.md

```diff
@@ -15,13 +15,11 @@ cluster.
 
 ## Prerequisites
 
-1. a Kubernetes v1.13+ is needed in order to support CSI Spec 1.0.
+1. Kubernetes v1.13+ is needed in order to support CSI Spec 1.0.
```

Comment:
This item is the only one formatted as a complete sentence. To maintain appropriate parallel structure, change this to just "Kubernetes v1.13+".